src/components/MapView/Legends/index.tsx

```diff
@@ -200,12 +270,44 @@ function LegendItem({
               <Typography variant="h5">{children}</Typography>
             </Grid>
           )}
+
+          {exposure && !analysisResult ? (
+            <AnalysisButton
+              variant="contained"
+              color="primary"
+              size="small"
+              onClick={runExposureAnalysis}
+            >
+              exposure analysis
```

Comment:
Is this cased correctly? Maybe `Exposure Analysis`, but up to you and how this is styled.

Reply:
There is already a css class which transforms text to uppercase. However, I made the change to keep code consistency. Included in 0992bfc3984b7e06a4d1c4a117db87a63bdb56a0